documentation/modules/ROOT/partials/modules/all-connectors/ref-connector-monitoring-streaming-metrics.adoc

```diff
@@ -14,19 +14,19 @@ The following table lists the streaming metrics that are available.
 
 |[[connectors-strm-metric-totalnumberofeventsseen_{context}]]<<connectors-strm-metric-totalnumberofeventsseen_{context}, `TotalNumberOfEventsSeen`>>
 |`long`
-|The total number of events that this connector has seen since the last start or metrics reset.
+|The total number of events that this connector has seen since the last start or metrics reset. (Calculated based on replication protocol events)
```

Comment:
```suggestion
|The total number of data change events reported by the source database since the last connector start, or since a metrics reset. 
Represents the data change workload for {prodname} to process.
```